tests/test_madlib.py

```diff
@@ -3,12 +3,12 @@
 
 
 def test_read_template_returns_stripped_string():
-    actual = read_template("assets/dark_and_stormy_night_template.txt")
+    actual = read_template("../assets/dark_and_stormy_night_template.txt")
```

Comment:
This path does not need to include `../`. When pytest or Python script is executed from the root of a project, it will begin searching for paths starting from the root. With this in mind the path should be `"assets/dark_and_stormy_night_template.txt"`